MAINTAINERS.md

```diff
@@ -126,7 +126,10 @@ git push origin <tagname>
 - [ ] ssh into `~zod` 
 - [ ] Check to ensure that nobody else is ssh'd into `~zod`, by running `screen -ls` and verifying no sessions are attached.
 - [ ] Attach to the screen session using `screen -x`
-- [ ] Install the contents of the `%kids` desk on `~marnec-dozzod-marzod` into our `%base`: `|merge %base ~marnec-dozzod-marzod %kids, =gem %only-that`
+- [ ] Ensure that the release candidate was correctly propagated through the prerelease moons
+  - Use `-read %z ~SHIP %DESK da+now /` to check desk hashes
+  - [ ] Run the above check on the following [SHIP DESK] pairs: [~marnec-dozzod-marzod %base], [~marnec-dozzod-marzod %kids], [~doznec-dozzod-marzod %base], [~doznec-dozzod-marzod %kids] — **they should all match**
+- [ ] Install the contents of the `%kids` desk on `~doznec-dozzod-marzod` into our `%base`: `|merge %base ~doznec-dozzod-marzod %kids, =gem %only-that`
```

Comment:
Thoughts @zalberico?

Reply:
the read command won't work if the clocks on the servers get out of sync for some reason, but otherwise should be fine (I typically just ssh into all of them myself)